blocks/api/factory.js

```diff
@@ -67,6 +68,48 @@ export function createBlock( name, blockAttributes = {} ) {
 }
 
 /**
+ * Returns an array of possible block transformations that could happen on the set of blocks received as argument.
+ *
+ * @param  {Array}  blocks Blocks array
+ * @return {Array}         Array of possible block transformations
+ */
+export function getPossibleBlockTransformations( blocks ) {
```

Comment:
Since it is nicely extracted into its own utility method, it would be even better to cover this functionality with unit tests :)

Reply:
>Since it is nicely extracted into its own utility method, it would be even better to cover this functionality with unit tests :)

Agree with this. Can we get some tests for this?

Reply:
tests were added :)